pasta_eln/gui.py

```diff
@@ -194,8 +195,9 @@ def execute(self, command: list[Any]) -> None:
 def mainGUI() -> tuple[Union[QCoreApplication, None], MainWindow]:
   """
     Main method and entry point for commands
-  Returns:
 
+  Returns:
```

Comment:
Actually it returns a tuple of app instance and main window..

Reply:
I will correct that